src/screens/RequestSwap/SelectSwapOption.tsx

```diff
@@ -12,6 +12,7 @@ interface SelectSwapOptionProps {
 export const SelectSwapOption = (props: SelectSwapOptionProps) => {
     const [selectedOption, setSelectedOption] = useState<number>(0);
     const iconDimensions = 50;
```

Comment:
the icons had turned invisible when i updated expo, so these just make the icons visible again